.github/workflows/contribs.yml

```
@@ -0,0 +1,23 @@
+name: Contribs
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  contribs:
+    runs-on: ubuntu-latest
+    name: Contribs
+    steps:
+      - name: Contribs
+        uses: carlescufi/action-contribs@main
+        with:
+          github-token: ${{ secrets.NCS_GITHUB_TOKEN }}
+          command: 'external'
+          messages: |
+                    Thank you for your contribution!
+                    It seems you are not a member of the nrfconnect GitHub organization.
+                    We are currently not accepting external contributions, but this may change soon.
+                    Please visit https://devzone.nordicsemi.com/ for further information.
+                    |
+                    The author of this Pull Request is now a member of the nrfconnect GitHub organization.
```

```suggestion
                    The author of this pull request is now a member of the nrfconnect GitHub organization.
```

Not sure where this comes in - is it a different case than the message before? Or do we want to say that the author of the PR has been changed to someone in the organization?

---

No, what I mean is that the author of the PR has been *added* to the organization.

---

OK, let's say this then. ;)